egs/fisher_swbd/s5/local/chain/run_tdnn_opgru_1a.sh

```diff
@@ -0,0 +1,269 @@
+#!/bin/bash
+# ./local/chain/compare_wer_general.sh tdnn_opgru_1a_sp
+# System                tdnn_opgru_1a_sp
```

Comment:
there is no comparison here with other topologies.  Can't you at least fake it based on the info that's checked into other scripts?
Please reorganize this directory to create a tuning directory, move the scripts into tuning/, and make soft links.

Reply:
Ok will do